src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinWorldServer_SimulationDistance.java

```diff
@@ -112,6 +112,7 @@ public MixinWorldServer_SimulationDistance() {
      */
     @WrapOperation(
             method = "func_147456_g",
+            remap = false,
```

Comment:
The `remap = false` should be inside `@At()`

Reply:
Mixins can work with incomplete mappings (`func_...` / `field_...`) in my experience. However, `canDoLightning` and `canDoRainSnowIce` are Forge methods, so these will cause problems. `remap` always also applies to all lower "levels", so your current code would still work. I just don't find it very elegant because it gives a wrong impression of what is a vanilla method and what isn't.